hexvexed-0.0.1.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="Hexagon game based  for haiku"
+DESCRIPTION="This is a hexagon based tile puzzle game forked from DarkWyrm's BeVexed \
+code to support 6 sides instead of four.   "
+
+HOMEPAGE="https://bitbucket.org/scottmc/hexvexed"
+COPYRIGHT="2009-2016 Scott McCreary"
+LICENSE="MIT"
+REVISION="3"
+SOURCE_URI="https://bitbucket.org/scottmc/hexvexed/get/68437a22c788.zip"
+SOURCE_DIR="scottmc-hexvexed-68437a22c788"
+CHECKSUM_SHA256="35e0904d91f15fefe801cdb7b9f9bdf4dc925de69f5b8818bec24cc4c22dc6d7"
+#PATCHES="
+#"
+ARCHITECTURES="x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+PROVIDES="
+	hexvexed = $portVersion
+	app:hexvexed = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+	
+ BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:make
+	cmd:g++
+	cmd:cp
+	"
+	
+BUILD()
+{
+	
+	make $jobArgs OBJ_DIR=objects
+	make $jobArgs OBJ_DIR=objects bindcatalogs
+}
+INSTALL() {
+	make install
+	}
```

Comment:
Oops, I lost my last comment. So here it is again:

Please define INSTALL this way:
```
NSTALL()
{
	make install
}
```
Please also have a look at the recipe for weather (haikuports/haiku-apps/weather/weather-0.1.1_git.recipe) to see how addAppDeskbarSymlink can be used.
BTW, if you are not building with the "-S" option of haikuporter, then please do so, because it can spot some common errors.